cluster/distvalidator.go

```diff
@@ -52,13 +55,21 @@ type distValidatorJSONv1x1 struct {
 	FeeRecipientAddress ethHex   `json:"fee_recipient_address,omitempty"`
 }
 
-// distValidatorJSONv1x2 is the json formatter of DistValidator for versions v1.2.0 and later.
-type distValidatorJSONv1x2 struct {
+// distValidatorJSONv1x2to5 is the json formatter of DistValidator for versions v1.2.0 to v1.5.0.
+type distValidatorJSONv1x2to5 struct {
 	PubKey              ethHex   `json:"distributed_public_key"`
 	PubShares           []ethHex `json:"public_shares,omitempty"`
 	FeeRecipientAddress ethHex   `json:"fee_recipient_address,omitempty"`
 }
 
+// distValidatorJSONv1x6 is the json formatter of DistValidator for versions v1.6.0 or later.
+type distValidatorJSONv1x6 struct {
+	PubKey              ethHex          `json:"distributed_public_key"`
+	PubShares           []ethHex        `json:"public_shares,omitempty"`
+	FeeRecipientAddress ethHex          `json:"fee_recipient_address,omitempty"`
```

Comment:
i think we should remove this field `FeeRecipientAddress` from here since it is not part of `DistValidator` and we don't use it

Reply:
yes indeed, should be removed, it isn't used.